recipes/ffmpeg/all/conanfile.py

```diff
@@ -347,6 +349,10 @@ def validate(self):
                 raise ConanInvalidConfiguration("FFmpeg '{}' option requires '{}' option to be enabled".format(
                     dependency, "' or '".join(features)))
 
+        if conan_version.major == 1 and is_msvc(self) and self.options.shared:
+            # Linking fails with "Argument list too long" for some reason on Conan v1
+            raise ConanInvalidConfiguration("MSVC shared build is not supported for Conan v1")
```

Comment:
I'm confused. It works fine with previous versions but not for 6.1.1, but this PR now prevents shared msvc with conan v1 for these versions also?

Reply:
It actually fails for all versions - I tested with disabling it just for v6.1.1, but it did not help and I force-pushed over it (sorry about that): https://github.com/conan-io/conan-center-index/compare/a4db9e764789ae8bca3f91b24d7e49e5dae2a276..395f7f6c00511abb9a88d2182d2c2e18959cef1c

Reply:
shared was building fine in this pass: https://github.com/conan-io/conan-center-index/pull/22195#issuecomment-2119346289